server/plaid.js

```diff
@@ -64,50 +64,96 @@ const noAccessTokenLogger = async (
-  createStripeToken: defaultLogger,
-  getInstitutions: noAccessTokenLogger,
-  getInstitutionById: noAccessTokenLogger,
-  searchInstitutionsByName: noAccessTokenLogger,
-  getCategories: noAccessTokenLogger,
-  createLinkToken: noAccessTokenLogger,
+  accountsBalanceGet: defaultLogger,
+  accountsGet: defaultLogger,
+  assetReportAuditCopyCreate: defaultLogger,
+  assetReportAuditCopyGet: noAccessTokenLogger,
+  assetReportAuditCopyRemove: noAccessTokenLogger,
+  assetReportCreate: noAccessTokenLogger,
+  assetReportFilter: noAccessTokenLogger,
+  assetReportGet: noAccessTokenLogger,
+  assetReportPdfGet: noAccessTokenLogger,
+  assetReportRefresh: noAccessTokenLogger,
+  assetReportRemove: noAccessTokenLogger,
+  authGet: defaultLogger,
+  bankTransferBalanceGet: noAccessTokenLogger,
+  bankTransferCancel: noAccessTokenLogger,
+  bankTransferCreate: defaultLogger,
+  bankTransferEventList: noAccessTokenLogger,
+  bankTransferEventSync: noAccessTokenLogger,
+  bankTransferGet: noAccessTokenLogger,
+  bankTransferList: noAccessTokenLogger,
+  bankTransferMigrateAccount: noAccessTokenLogger,
+  categoriesGet: noAccessTokenLogger,
+  depositSwitchAltCreate: noAccessTokenLogger,
+  depositSwitchCreate: noAccessTokenLogger,
+  depositSwitchGet: noAccessTokenLogger,
+  depositSwitchTokenCreate: noAccessTokenLogger,
+  employersSearch: noAccessTokenLogger,
+  transactionsGet: defaultLogger,
+  identityGet: defaultLogger,
+  institutionsGet: noAccessTokenLogger,
+  institutionsGetById: noAccessTokenLogger,
+  institutionsSearch: noAccessTokenLogger,
+  investmentsHoldingsGet: defaultLogger,
+  investmentsTransactionsGet: defaultLogger,
+  itemAccessTokenInvalidate: defaultLogger,
+  itemCreatePublicToken: defaultLogger,
+  itemGet: defaultLogger,
+  itemPublicTokenExchange: noAccessTokenLogger,
+  itemRemove: defaultLogger,
+  itemWebhookUpdate: defaultLogger,
+  liabilitiesGet: defaultLogger,
+  linkTokenCreate: noAccessTokenLogger,
+  linkTokenGet: noAccessTokenLogger,
+  paymentInitiationPaymentCreate: noAccessTokenLogger,
+  paymentInitiationPaymentGet: noAccessTokenLogger,
+  paymentInitiationPaymentList: noAccessTokenLogger,
+  paymentInitiationRecipientCreate: noAccessTokenLogger,
+  paymentInitiationRecipientGet: noAccessTokenLogger,
+  paymentInitiationRecipientList: noAccessTokenLogger,
+  processorAuthGet: noAccessTokenLogger,
+  processorBalanceGet: noAccessTokenLogger,
+  processorIdentityGet: noAccessTokenLogger,
+  processorStripeBankAccountTokenCreate: defaultLogger,
+  processorTokenCreate: defaultLogger,
+  sandboxBankTransferFireWebhook: noAccessTokenLogger,
+  transactionsGet: defaultLogger,
+  sandboxBankTransferSimulate: noAccessTokenLogger,
+  transactionsGet: defaultLogger,
+  transactionsGet: defaultLogger,
+  sandboxPublicTokenCreate: noAccessTokenLogger,
+  transactionsGet: defaultLogger,
+  transactionsRefresh: defaultLogger,
+  webhookVerificationKeyGet: noAccessTokenLogger,
+  transactionsGet: defaultLogger,
```

Comment:
this looks like you added a bunch of functions that are not actually used in this app. Is this just so its there for future use for the sample use case apps?

Reply:
yes, this is there for future use...this big list was also included in the old Pattern.